packs/typeform/actions/get_results.py

```diff
@@ -0,0 +1,40 @@
+import urllib
+import httplib
+import requests
+
+from st2actions.runners.pythonrunner import Action
+
+__all__ = [
+    'TypeformAction'
+]
+
+
+class TypeformAction(Action):
+
+    def run(self, form_id, api_key, completed=True):
+        api_key = api_key if api_key else self.config['api_key']
+        completed = str(completed).lower()
+        url = "https://api.typeform.com/v0/form/%s?key=%s&completed=%s" % \
+            (form_id, api_key, completed)
+
+        headers = {}
+        headers['Content-Type'] = 'application/x-www-form-urlencoded'
+
+        params = {"key": api_key,
+                  "completed": completed
+                  }
+
+        data = urllib.urlencode(params)
+        response = requests.get(url=url,
+                                headers=headers, params=data)
+
+        if response.status_code == httplib.OK:
```

Comment:
I want to make sure I am returning response.json() so we can parse it properly.  I see what you are saying but returning response.json() on a 200 seemed the best way to make sure we got the structured data.